src/browser/actions/makeStorageActions.js

```diff
@@ -16,14 +16,17 @@ const getStorage = async ({ net, scriptHash, key, encode = true }) => {
 
   const endpoint = await api.loadBalance(api.getRPCEndpointFrom, { net });
   const { result } =
-    await rpc.Query.getStorage(scriptHash, encode ? u.str2hexstring(key) : key).execute(endpoint);
-  return result;
+    await rpc.Query.getStorage(scriptHash, encodeInput
+      ? u.str2hexstring(key)
+      : key)
+      .execute(endpoint);
+  return decodeOutput ? u.hexstring2str(result) : result;
 };
 
 export default function makeStorageActions(sessionId, requestId) {
   const id = generateDAppActionId(sessionId, `${ID}-${requestId}`);
 
-  return createActions(id, ({ net, scriptHash, key, encode }) => async () => {
-    return getStorage({ net, scriptHash, key, encode });
+  return createActions(id, ({ net, scriptHash, key, encodeInput, decodeOutput }) => async () => {
```

Comment:
Similar suggestion here.  Since this is the "public-facing" function (the one that is exported), I think we should specify the defaults here rather than on line 8.

Reply:
What's the benefit of this? Can it be seen on autocomplete or such? :P

Reply:
It's just to ensure we're passing the intended values, nothing more.